src/Entity/Slot.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+namespace Drupal\acquia_lift\Entity;
+
+use Acquia\LiftClient\Entity\Visibility;
+use Drupal\acquia_lift\AcquiaLiftException;
+use Drupal\acquia_lift\SlotInterface;
+use Drupal\Core\Config\Entity\ConfigEntityBase;
+use Drupal\Core\Entity\EntityStorageInterface;
+
+/**
+ * Defines the Drupal created slots.
+ *
+ * @ConfigEntityType(
+ *   id = "acquia_lift_slot",
```

Comment:
I was not able to find Drupal documentation yesterday, but I can try to mount arguments:

* The closest documentation I can find is [config#format](https://www.drupal.org/coding-standards/config#format). It attempts to dot-split all the layers.
* On the other hand, if we have to go for "acquia_lift_slot" id, then the current "config/schema/acquia_lift.slot.yml" would be wrong, it should be "config/schema/acquia_lift_slot.yml"; its line 1 needs to be "acquia_lift_slot.*". Also, "acquia_lift.settings" would then need to be renamed to "acquia_lift_settings" to align with "acquia_lift_slot". This in general would be against the commonly observed schema.yml patterns.